--- Syntax.original	2021-01-17 22:19:27.631342941 +0100
+++ Syntax	2021-01-17 22:20:26.941502988 +0100
@@ -214,7 +214,7 @@
 file control$ Debian\scontrol\sfile
 include debian-control.syntax
 
-file (rules|rocks)$ Debian\srules
+file (rocks)$ Debian\srules
 include makefile.syntax
 
 file .\*changelog$ Debian\schangelog\sfile
@@ -298,5 +298,20 @@
 file ..\*\\.proto$ Protobuf\sFile
 include protobuf.syntax
 
+file ..\*\\.(items)$ openHAB\sItems
+include openhab-items.syntax
+
+file ..\*\\.(sitemap)$ openHAB\sSitemap
+include openhab-sitemap.syntax
+
+file ..\*\\.(persist)$ openHAB\sPersistence
+include openhab-persist.syntax
+
+file ..\*\\.(rules)$ openHAB\sRules
+include openhab-rules.syntax
+
+file ..\*\\.(script)$ openHAB\sScript
+include openhab-script.syntax
+
 file .\* unknown
 include unknown.syntax
